download_and_install/download_and_install.md

```diff
@@ -0,0 +1,52 @@
+Download and Install
+====================
+
+Pre-built binaries
+------------------
+
+OLA is available pre-built on many platforms:
+
+### Debian/Ubuntu (including Raspberry Pi)
+
+Raspberry Pi users may want to see the [Raspberry Pi Quick-Start Guide](ola_on_raspberry_pi)
+
+Ubuntu users will need the `universe` repository enabled.
+
+    sudo apt-get update
+    sudo apt-get install ola
+
+### OpenSUSE
+
+Available from [multimedia:libs](https://build.opensuse.org/package/show/multimedia:libs/ola):
+
+    sudo zypper addrepo https://download.opensuse.org/repositories/multimedia:/libs/<VERSION> multimedia_libs
+    sudo zypper install ola
+
+If this is the first time software is installed from multimedia:liba, you may need to accept their package signing key.
+
+### Gentoo
+
+Gentoo packages are in progress [here](https://github.com/gentoo/gentoo/pull/15017).
```

Comment:
I've updated this yesterday to a new PR: https://github.com/gentoo/gentoo/pull/19420